[BUG] some typos and warnings found by static check

**What is your environment(Kubernetes version, Fluid version, etc.)**

**Describe the bug**

Fix some typos and warnings found by static check

https://staticcheck.io/


```
staticcheck ./...
pkg/controllers/runtime_controller.go:66:31: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/delete_volume.go:77:11: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/hcfs.go:62:15: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/health_check.go:137:9: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/health_check.go:227:9: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/health_check.go:308:9: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/operations/base.go:258:9: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/operations/base.go:264:9: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/operations/base.go:303:9: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/operations/base.go:309:9: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/operations/base.go:373:9: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/operations/base_test.go:44:7: receiver name should not be an underscore, omit the name if it is unused (ST1006)
pkg/ddc/alluxio/operations/base_test.go:70:7: receiver name should not be an underscore, omit the name if it is unused (ST1006)
pkg/ddc/alluxio/operations/base_test.go:74:7: receiver name should not be an underscore, omit the name if it is unused (ST1006)
pkg/ddc/alluxio/operations/cached.go:44:20: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/operations/cached.go:55:9: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/shutdown.go:135:9: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/shutdown.go:135:9: error strings should not end with punctuation or a newline (ST1005)
pkg/ddc/alluxio/transform.go:28:9: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/ufs_internal.go:71:9: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/ufs_internal.go:115:10: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/utils.go:277:10: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/utils.go:282:9: error strings should not be capitalized (ST1005)
pkg/ddc/alluxio/worker.go:51:10: error strings should not be capitalized (ST1005)
pkg/ddc/factory.go:44:9: error strings should not be capitalized (ST1005)
pkg/ddc/jindo/transform.go:12:9: error strings should not be capitalized (ST1005)
pkg/utils/helm/helm.go:103:28: error strings should not be capitalized (ST1005)
pkg/utils/helm/helm.go:145:14: error strings should not be capitalized (ST1005)
pkg/utils/helm/helm.go:149:14: error strings should not be capitalized (ST1005)
pkg/utils/home.go:39:5: don't use Yoda conditions (ST1017)
pkg/utils/kubeclient/pod.go:21:2: package "k8s.io/api/core/v1" is being imported more than once (ST1019)
	pkg/utils/kubeclient/pod.go:22:2: other import of "k8s.io/api/core/v1"
pkg/utils/kubectl/configmap.go:85:20: error strings should not be capitalized (ST1005)
```

**What you expect to happen:**

**How to reproduce it**

**Additional Information**

--- a/pkg/controllers/runtime_controller.go
+++ b/pkg/controllers/runtime_controller.go
@@ -63,7 +63,7 @@ func (r *RuntimeReconciler) ReconcileInternal(ctx cruntime.ReconcileRequestConte
 	// 1.Reconcile runtime
 	runtime := ctx.Runtime
 	if runtime == nil {
-		return utils.RequeueIfError(fmt.Errorf("Failed to find the runtime"))
+		return utils.RequeueIfError(fmt.Errorf("failed to find the runtime"))
 	}
 
 	engine, err := r.implement.GetOrCreateEngine(ctx)

--- a/pkg/ddc/alluxio/delete_volume.go
+++ b/pkg/ddc/alluxio/delete_volume.go
@@ -74,10 +74,10 @@ func (e *AlluxioEngine) deleteFusePersistentVolume() (err error) {
 		}
 
 		if found {
-			return fmt.Errorf("The PV %s is not cleaned up",
+			return fmt.Errorf("the PV %s is not cleaned up",
 				e.runtime.Name)
 		} else {
-			e.Log.Info("The PV is deleted successfully", "name", e.runtime.Name)
+			e.Log.Info("the PV is deleted successfully", "name", e.runtime.Name)
 		}
 	}
 

--- a/pkg/ddc/alluxio/hcfs.go
+++ b/pkg/ddc/alluxio/hcfs.go
@@ -59,7 +59,7 @@ func (e *AlluxioEngine) queryHCFSEndpoint() (endpoint string, err error) {
 	}
 
 	if svc == nil {
-		e.Log.Error(fmt.Errorf("Failed to find the svc %s in %s", e.name, e.namespace), "Failed to find the svc, it's nil")
+		e.Log.Error(fmt.Errorf("failed to find the svc %s in %s", e.name, e.namespace), "failed to find the svc, it's nil")
 		return
 	}
 

--- a/pkg/ddc/alluxio/health_check.go
+++ b/pkg/ddc/alluxio/health_check.go
@@ -134,7 +134,7 @@ func (e *AlluxioEngine) checkMasterHealthy() (err error) {
 	}
 
 	if !healthy {
-		err = fmt.Errorf("the master %s in %s is not ready. the expected number is %d, the actual number is %d",
+		err = fmt.Errorf("the master %s in %s is not ready. The expected number is %d, the actual number is %d",
 			master.Name,
 			master.Namespace,
 			master.Status.Replicas,

--- a/pkg/ddc/alluxio/health_check.go
+++ b/pkg/ddc/alluxio/health_check.go
@@ -134,7 +134,7 @@ func (e *AlluxioEngine) checkMasterHealthy() (err error) {
 	}
 
 	if !healthy {
-		err = fmt.Errorf("The master %s in %s is not ready. The expected number is %d, the actual number is %d",
+		err = fmt.Errorf("the master %s in %s is not ready. the expected number is %d, the actual number is %d",
 			master.Name,
 			master.Namespace,
 			master.Status.Replicas,
@@ -224,7 +224,7 @@ func (e *AlluxioEngine) checkWorkersHealthy() (err error) {
 	}
 
 	if !healthy {
-		err = fmt.Errorf("The daemonset %s in %s are not ready, the unhealthy number %d",
+		err = fmt.Errorf("the daemonset %s in %s are not ready, the unhealthy number %d",
 			workers.Name,
 			workers.Namespace,
 			workers.Status.NumberUnavailable)
@@ -305,7 +305,7 @@ func (e *AlluxioEngine) checkFuseHealthy() (err error) {
 	}
 
 	if !healthy {
-		err = fmt.Errorf("The daemonset %s in %s are not ready, the unhealthy number %d",
+		err = fmt.Errorf("the daemonset %s in %s are not ready, the unhealthy number %d",
 			fuses.Name,
 			fuses.Namespace,
 			fuses.Status.UpdatedNumberScheduled)

--- a/pkg/ddc/alluxio/operations/base.go
+++ b/pkg/ddc/alluxio/operations/base.go
@@ -255,13 +255,13 @@ func (a AlluxioFileUtils) Du(alluxioPath string) (ufs int64, cached int64, cache
 	str := strings.Split(stdout, "\n")
 
 	if len(str) != 2 {
-		err = fmt.Errorf("Failed to parse %s in Du method", str)
+		err = fmt.Errorf("failed to parse %s in Du method", str)
 		return
 	}
 
 	data := strings.Fields(str[1])
 	if len(data) != 4 {
-		err = fmt.Errorf("Failed to parse %s in Du method", data)
+		err = fmt.Errorf("failed to parse %s in Du method", data)
 		return
 	}
 
@@ -300,13 +300,13 @@ func (a AlluxioFileUtils) Count(alluxioPath string) (fileCount int64, folderCoun
 	str := strings.Split(stdout, "\n")
 
 	if len(str) != 2 {
-		err = fmt.Errorf("Failed to parse %s in Count method", str)
+		err = fmt.Errorf("failed to parse %s in Count method", str)
 		return
 	}
 
 	data := strings.Fields(str[1])
 	if len(data) != 3 {
-		err = fmt.Errorf("Failed to parse %s in Count method", data)
+		err = fmt.Errorf("failed to parse %s in Count method", data)
 		return
 	}
 
@@ -370,7 +370,7 @@ func (a AlluxioFileUtils) exec(command []string, verbose bool) (stdout string, s
 	case <-ch:
 		a.log.V(1).Info("execute in time", "command", command)
 	case <-ctx.Done():
-		err = fmt.Errorf("Timeout when executing %v", command)
+		err = fmt.Errorf("timeout when executing %v", command)
 	}
 
 	return

--- a/pkg/ddc/alluxio/operations/base_test.go
+++ b/pkg/ddc/alluxio/operations/base_test.go
@@ -41,7 +41,7 @@ const (
 // a empty logger just for testing ...
 type NullLogger struct{}
 
-func (_ NullLogger) Info(_ string, _ ...interface{}) {
+func (log NullLogger) Info(_ string, _ ...interface{}) {
 	// Do nothing.
 }
 
@@ -67,11 +67,11 @@ func TestLoadMetaData(t *testing.T) {
 	}
 }
 
-func (_ NullLogger) Enabled() bool {
+func (log NullLogger) Enabled() bool {
 	return false
 }
 
-func (_ NullLogger) Error(_ error, _ string, _ ...interface{}) {
+func (log NullLogger) Error(_ error, _ string, _ ...interface{}) {
 	// Do nothing.
 }
 

--- a/pkg/ddc/alluxio/operations/cached.go
+++ b/pkg/ddc/alluxio/operations/cached.go
@@ -41,7 +41,7 @@ func (a AlluxioFileUtils) CachedState() (cached int64, err error) {
 		if strings.Contains(s, "Used Capacity:") {
 			values := strings.Fields(s)
 			if len(values) == 0 {
-				return cached, fmt.Errorf("Failed to parse %s", s)
+				return cached, fmt.Errorf("failed to parse %s", s)
 			}
 			cached, err = units.RAMInBytes(values[len(values)-1])
 			if err != nil {
@@ -52,7 +52,7 @@ func (a AlluxioFileUtils) CachedState() (cached int64, err error) {
 	}
 
 	if !found {
-		err = fmt.Errorf("Failed to find the cache in output %v", stdout)
+		err = fmt.Errorf("failed to find the cache in output %v", stdout)
 	}
 
 	return

--- a/pkg/ddc/alluxio/shutdown.go
+++ b/pkg/ddc/alluxio/shutdown.go
@@ -132,7 +132,7 @@ func (e *AlluxioEngine) cleanupCache() (err error) {
 	// 	return fmt.Errorf("The remaining cached is not cleaned up, it still has %d", cached)
 	// }
 
-	return fmt.Errorf("The remaining cached is not cleaned up, check again.")
+	return fmt.Errorf("the remaining cached is not cleaned up, check again")
 }
 
 // cleanAll cleans up the all

--- a/pkg/ddc/alluxio/transform.go
+++ b/pkg/ddc/alluxio/transform.go
@@ -25,7 +25,7 @@ import (
 
 func (e *AlluxioEngine) transform(runtime *datav1alpha1.AlluxioRuntime) (value *Alluxio, err error) {
 	if runtime == nil {
-		err = fmt.Errorf("The alluxioRuntime is null")
+		err = fmt.Errorf("the alluxioRuntime is null")
 		return
 	}
 

--- a/pkg/ddc/alluxio/ufs_internal.go
+++ b/pkg/ddc/alluxio/ufs_internal.go
@@ -68,7 +68,7 @@ func (e *AlluxioEngine) shouldMountUFS() (should bool, err error) {
 	ready := fileUitls.Ready()
 	if !ready {
 		should = false
-		err = fmt.Errorf("The UFS is not ready")
+		err = fmt.Errorf("the UFS is not ready")
 		return should, err
 	}
 
@@ -112,7 +112,7 @@ func (e *AlluxioEngine) mountUFS() (err error) {
 
 	ready := fileUitls.Ready()
 	if !ready {
-		return fmt.Errorf("The UFS is not ready")
+		return fmt.Errorf("the UFS is not ready")
 	}
 
 	// Iterate all the mount points, do mount if the mount point is not Fluid-native(e.g. Hostpath or PVC)

--- a/pkg/ddc/alluxio/utils.go
+++ b/pkg/ddc/alluxio/utils.go
@@ -274,12 +274,12 @@ func (e *AlluxioEngine) getAvaliablePort() (allocatedPorts []int, err error) {
 		}
 
 		if !found {
-			err = fmt.Errorf("All avaliable port from %d to %d are allocated", AUTO_SELECT_PORT_MIN, AUTO_SELECT_PORT_MAX)
+			err = fmt.Errorf("all avaliable port from %d to %d are allocated", AUTO_SELECT_PORT_MIN, AUTO_SELECT_PORT_MAX)
 		}
 	}
 
 	if len(allocatedPorts) != portNum {
-		err = fmt.Errorf("Can`t allocate enough port, got %d but expect %d", len(allocatedPorts), portNum)
+		err = fmt.Errorf("can`t allocate enough port, got %d but expect %d", len(allocatedPorts), portNum)
 	}
 
 	return

--- a/pkg/ddc/alluxio/worker.go
+++ b/pkg/ddc/alluxio/worker.go
@@ -48,7 +48,7 @@ func (e *AlluxioEngine) SetupWorkers() (err error) {
 		"currentReplicas", currentReplicas)
 
 	if currentReplicas == 0 {
-		return fmt.Errorf("The number of the current workers which can be scheduled is 0")
+		return fmt.Errorf("the number of the current workers which can be scheduled is 0")
 	}
 
 	// 2. Update the status

--- a/pkg/ddc/factory.go
+++ b/pkg/ddc/factory.go
@@ -41,7 +41,7 @@ func CreateEngine(id string, ctx cruntime.ReconcileRequestContext) (engine base.
 	if buildeFunc, found := buildFuncMap[ctx.RuntimeType]; found {
 		engine, err = buildeFunc(id, ctx)
 	} else {
-		err = fmt.Errorf("Failed to build the engine due to the type %s is not found", ctx.NamespacedName)
+		err = fmt.Errorf("failed to build the engine due to the type %s is not found", ctx.NamespacedName)
 	}
 
 	return

--- a/pkg/ddc/jindo/transform.go
+++ b/pkg/ddc/jindo/transform.go
@@ -9,7 +9,7 @@ import (
 
 func (e *JindoEngine) transform(runtime *datav1alpha1.JindoRuntime) (value *Jindo, err error) {
 	if runtime == nil {
-		err = fmt.Errorf("The jindoRuntime is null")
+		err = fmt.Errorf("the jindoRuntime is null")
 		return
 	}
 

--- a/pkg/utils/helm/helm.go
+++ b/pkg/utils/helm/helm.go
@@ -100,7 +100,7 @@ func GenerateHelmTemplate(name string, namespace string, valueFileName string, c
 	out, err := cmd.CombinedOutput()
 	fmt.Printf("%s", string(out))
 	if err != nil {
-		return templateFileName, fmt.Errorf("Failed to execute %s, %v with %v", binary, args, err)
+		return templateFileName, fmt.Errorf("failed to execute %s, %v with %v", binary, args, err)
 	}
 
 	// // 6. clean up the value file if needed
@@ -142,11 +142,11 @@ func GetChartVersion(chart string) (version string, err error) {
 
 	lines := strings.Split(string(out), "\n")
 	if len(lines) == 0 {
-		return "", fmt.Errorf("Failed to find version when executing %s, result is %s", args, out)
+		return "", fmt.Errorf("failed to find version when executing %s, result is %s", args, out)
 	}
 	fields := strings.Split(lines[0], ":")
 	if len(fields) != 2 {
-		return "", fmt.Errorf("Failed to find version when executing %s, result is %s", args, out)
+		return "", fmt.Errorf("failed to find version when executing %s, result is %s", args, out)
 	}
 
 	version = strings.TrimSpace(fields[1])

--- a/pkg/utils/home.go
+++ b/pkg/utils/home.go
@@ -36,7 +36,7 @@ func Home() (string, error) {
 
 	// cross compile support
 
-	if "windows" == runtime.GOOS {
+	if runtime.GOOS == "windows" {
 		return homeWindows()
 	}
 

--- a/pkg/utils/kubectl/configmap.go
+++ b/pkg/utils/kubectl/configmap.go
@@ -82,7 +82,7 @@ func SaveConfigMapToFile(name string, key string, namespace string) (fileName st
 	fmt.Printf("%s", string(out))
 
 	if err != nil {
-		return fileName, fmt.Errorf("Failed to execute %s, %v with %v", "kubectl", args, err)
+		return fileName, fmt.Errorf("failed to execute %s, %v with %v", "kubectl", args, err)
 	}
 	return fileName, err
 }

